profiles/base.yaml

```diff
@@ -1037,6 +1037,31 @@ boggle_junk:
 - rose
 - wallet
 
+maze_junk:
+- earrings
+- puppet
+- poppet
+- ribbon
+- charm
+- wheelbarrow
+- garter
+- pear foil
+- mouse
+- radish
+- tankard
+- girdle
+- pin
+- statue
+- farmer hat
+- armure tunic
+- pipe
+- blossom
+- muffin
+- brocade pouch
+- socks
+
+# Define a list of containers like you would for storage_containers.  All the bags you could shove stuff into for the cornmaze 
+cornmaze_containers:
```

Comment:
Would you remove this from base as you have it in base-empty?

Reply:
nope it should still be here too